pkg/scanner/scan.go

```diff
@@ -105,7 +106,7 @@ func NewScanner(driver Driver, ar artifact.Artifact) Scanner {
 }
 
 // ScanArtifact scans the artifacts and returns results
-func (s Scanner) ScanArtifact(ctx context.Context, options types.ScanOptions) (types.Report, error) {
+func (s Scanner) ScanArtifact(ctx context.Context, options types.ScanOptions, ac cache.ArtifactCache) (types.Report, error) {
```

Comment:
`s.artifact` already has cache. We don't have to pass it here.

Reply:
sure, removed.